media/css/mozorg/home/home.scss

```diff
@@ -140,6 +140,20 @@ body {
     }
 }
 
+html[dir="rtl"] .masthead-nav-main {
```

Comment:
Yes, good suggestion. Will move. (Still getting comfortable with the pebbles structure...)